src/uct/ib/rc/base/rc_iface.h

```diff
@@ -494,9 +494,12 @@ uct_rc_iface_add_cq_credits(uct_rc_iface_t *iface, uint16_t cq_credits)
 static UCS_F_ALWAYS_INLINE uct_rc_iface_send_op_t*
 uct_rc_iface_get_send_op(uct_rc_iface_t *iface)
 {
-    uct_rc_iface_send_op_t *op;
-    op = iface->tx.free_ops;
+    uct_rc_iface_send_op_t *op = iface->tx.free_ops;
+
+    ucs_assert(op != NULL);
```

Comment:
remove this assert

Reply:
it would be helpful to find errors not by Valgrind, but by assertion

Reply:
wouldn't the next line segfault anyway?

Reply:
ok, removed